docs/usage/options.mdx

```diff
@@ -126,6 +126,11 @@ Use the Playground to edit and play around with some of these options!
 				"#8c564b",
 			],
 			enableTooltip: true,
+			tooltipProps: {
+				allowHTML: true,
+				arrow: false,
+				placement: "bottom",
+			},
```

Comment:
let's sort these alphabetically by key value to be consistent with the existing code.  Same for the other code changes in other files in this diff.

Reply:
Ah, I didn't follow this comment because the quoted lines are in order already, and there was one prop which was out of order and confusing me (https://github.com/chrisrzhou/react-wordcloud/blame/master/docs/usage/options.mdx#L129).  Makes sense now - I'll make it happen